lib/internal/bootstrap/node.js

```diff
@@ -387,6 +387,16 @@ function setupGlobalProxy() {
   });
 }
 
+function reportError(error) {
+  if (arguments.length === 0) {
+    const { codes } = require('internal/errors');
+    throw new codes.ERR_MISSING_ARGS('error');
+  }
+  queueMicrotask(() => {
+    throw error;
```

Comment:
should this generate a stack trace if no error was provided? mainly thinking of what happens / how a user could back track the problem from `reportError()`.

Reply:
Maybe emit a warning (synchronously) with a stack trace if a non-Error was passed?

Reply:
sure